components/StyledInputField.js

```diff
@@ -14,6 +15,14 @@ const PrivateIconWithSpace = () => (
   </React.Fragment>
 );
 
+// eslint-disable-next-line react/prop-types
+const QuestionMarkIconWithSpace = ({ helpText, helpIconColor, helpIconSize }) => (
+  <React.Fragment>
+    &nbsp;
+    <Question title={helpText} size={helpIconSize} color={helpIconColor} />
```

Comment:
Shouldn't we use a `StyledTooltip` there?

Reply:
Changed it to `StyledTooltip`. I changed both instances in the page so that it's consistent.